src/konfo_backend/index/toteutus.clj

```diff
@@ -22,25 +22,30 @@
   [oid draft?]
   (let [toteutus (get-source index oid)]
     (when (allowed-to-view toteutus draft?)
-      (cond-> toteutus
-              (some? (:hakutiedot toteutus)) (map-allowed-to-view-hakutiedot draft?)))))
+      (-> toteutus
+          (map-allowed-to-view-hakutiedot draft?)
+          (as-> t
+                (assoc t :hakuAuki (toteutus-haku-kaynnissa? t))
+            (assoc t :hakutiedot (with-is-haku-auki (:hakutiedot t))))))))
```

Comment:
Onkos tässä sisennys vähän hassusti? Ton jälkimmäisen associn pitäis olla samalla tasolla ylemmän kanssa?

Reply:
Korjasin tämän